spec/acceptance/unattended_upgrades_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper_acceptance'
+
+describe 'unattended_upgrades with default settings' do
+  context 'default parameters' do
+    it 'works with defaults' do
+      pp = 'include unattended_upgrades'
+      # Run it twice and test for idempotency
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+    end
+  end
```

Comment:
This probably needs to go in another PR, but I'll review it here. Is this a good case for https://github.com/voxpupuli/voxpupuli-acceptance/#an-idempotent-resource?

Reply:
Done, thanks!